SALib/analyze/sobol.py

```diff
@@ -8,14 +8,18 @@
 from . import common_args
 from ..util import read_param_file
 
+from multiprocessing import Pool, cpu_count
+from functools import partial
+
+EMPTY = -1

```

Comment:
I'm not too keen on this variable.  What does it mean?  An inline comment could help clarify what its for.  Also doesn't meet PEP standards.  Boo to ALLCAPS ;)